docs/DMS/DMS-AUTH.md

```diff
@@ -37,62 +37,89 @@ update is authorized.
+access rights defined in the API details context. Specifically, the system
+verifies if the client has access to the requested educational organization or
+namespace prefix.
 
 ### 6. Check incoming request against Claim Set details
 
-The incoming resource and action will be compared to the resource claims in the claim set. If the resource name is present in the resource claims list, the action will be validated according to the specified authorization strategies. Finally, the resource data will be checked to ensure compliance with the authorization strategy rule sets. If all verification steps pass, the requested action will proceed.
+The incoming resource and action will be compared to the resource claims in the
+claim set. If the resource name is present in the resource claims list, the
+action will be validated according to the specified authorization strategies.
+Finally, the resource data will be checked to ensure compliance with the
+authorization strategy rule sets. If all verification steps pass, the requested
+action will proceed.
 
 ## Namespace based authorization on DMS
 
 ### 1. Authenticating and Authorizing the Client on DMS
 
-The client provides its client ID and secret to the oauth/token endpoint on DMS. The system then authenticates these credentials using a third-party authentication provider(`Keycloak`), which returns a token upon successful authentication.
+The client provides its client ID and secret to the oauth/token endpoint on DMS.
+The system then authenticates these credentials using a third-party
+authentication provider(`Keycloak`), which returns a token upon successful
+authentication.
 
 ### 2. Token Interpreter
 
-The token interpreter decodes the received token to extract basic authorization metadata, including the Client id, Claim set name, Education Organization ids, Namespace prefixes etc...
+The token interpreter decodes the received token to extract basic authorization
+metadata, including the Client id, Claim set name, Education Organization ids,
+Namespace prefixes etc...
 
 ### 3. Authorization Context
 
-An authorization context is established, including the client's basic authorization details (Client id, Claim set name, Education Organization ids, Namespace prefixes etc..).
-This context data is then cached to streamline further requests.
+An authorization context is established, including the client's basic
+authorization details (Client id, Claim set name, Education Organization ids,
+Namespace prefixes etc..). This context data is then cached to streamline
+further requests.
 
 ### 4. Security Metadata Cache
 
-In the ODS API, the entire security metadata is cached. Caching only the claim set data may be insufficient, as parent entity details could also be required for authorization.
+In the ODS API, the entire security metadata is cached. Caching only the claim
+set data may be insufficient, as parent entity details could also be required
+for authorization.
```

Comment:
Yes, we may need to cache parent information, but we won't need to worry about that until we're designing for relationship-based authorization in a future ticket.